server/methods/setStatusMessage.coffee

```diff
@@ -0,0 +1,12 @@
+Meteor.methods
+	setStatusMessage: (status, message) ->
+		userId = Meteor.userId()
+		if not userId
+			throw new Meteor.Error('invalid-user', "[methods] toogleFavorite -> Invalid user")
+
+		console.log '[methods] setStatusMessage -> '.green, 'userId:', Meteor.userId(), 'arguments:', arguments
+
+		user = Meteor.user()
+		setField = { statusMessages:user.profile.statusMessages}
+		setField.statusMessages[status] = message
```

Comment:
errors out when user.profile.statusMessages is undefined.  e.g. when no status message has ever been set.